### DISCUSSION TITLE
Hi , can you help me with this code

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#181] [Q&A]
How to put data array in horizontal line.

And please put with click function too .

```javascript
                        const dataArray = [
  { text: '0' },
  { text: '1' },
  { text: '2' },
  { text: '3' },
  { text: '4' },
  { text: '5' },
  { text: '6' },
  { text: '7' },
  { text: '8' },
  { text: '9' }
]

 hmUI.createWidget(hmUI.widget.CYCLE_IMAGE_TEXT_LIST, {
  x: 1,
  y: 0,
  w: 300,
  h: 100,
  data_array: dataArray,
  data_size: 10,
  item_height: 30,
  item_text_color: 0xffffff,
  item_text_size: 20
})

```

**Comment by MyoungXUE** [comment]
Hello, I do not understand what kind of effect you are talking about, can you provide more detailed text, or with pictures to describe in more detail?

**Reply by alexmathew11** [reply]
Hello by default data array is shown vertically but how to show horizontaly?

And can you add click functions too?

**Reply by MyoungXUE** [reply]
This widget cannot be scrolled horizontally, so if you need to add a click event, use the `item_click_func` property.

I will add examples of this section to the sample Mini Program as soon as possible.
